packages/grafana-ui/src/components/IconButton/IconButton.story.tsx

```diff
@@ -101,31 +143,30 @@ export const ExamplesBackground = () => {
   );
 };
 
-const RenderBackgroundScenario = ({ background }: ScenarioProps) => {
+export const ExamplesWithoutTooltip = (args: BasePropsWithAriaLabel) => {
```

Comment:
Ok... 👀  I know I was the one who suggested this, but now after seeing this story and how we say we do not recommend using this, maybe it is not necessary? 😅  What do you think?

Reply:
I find it much more handy for tooltip in comparison to aria-label. For tooltip it helps to see how it would look like when using different descriptions. For aria-label it doesn't make much sense.
So I will remove the control for aria-label but I would like to keep it for tooltip.

Reply:
Yep I'd say remove the story that doesnt have tooltips completely 🤷